db/sg_replicate_conflict_resolver.go

```diff
@@ -51,10 +51,13 @@ const (
 
 // Conflict is the input to all conflict resolvers.  LocalDocument and RemoteDocument
 // are expected to be document bodies with metadata injected into the body following
-// the same approach used for doc and oldDoc in the Sync Function
+// the same approach used for doc and oldDoc in the Sync Function. LocalHLV and RemoteHLV
+// will be the documents HLVs, these are needed for HLV ready conflict resolution.
```

Comment:
```suggestion
// will be the documents HLVs.
```

Document if these can be nil. In the case of legacy rev with v4 protocol, will this get run? What about incoming legacy rev with existing hlv?